docs/test_instructions_per_dependency_update.md

```diff
@@ -0,0 +1,859 @@
+  - I recommend using a `versionCode` of `1000000001` and above.
+  - I recommend using an explicit `versionName` like `deps-update-google-play-services-20-4-1`
+    (similar to the name of this branch itself)
+- Build the release `.aab` locally using this command: `./gradlew bundleWordPressVanillaRelease`
+- Upload the locally generated `.aab`, found within the `WordPress/build/outputs/bundle/wordpressVanillaRelease/`
+  build folder, and into Google Play Console (`WordPress - Website Builder`) via:
+  `App Bundle Explorer` > `Upload new version` (top-right)
+  - Wait for the upload to complete, then via this newly uploaded app version, navigation to the
+    `Downloads`, on the `Assets` section, find the `Signed, universal APK` and click the `Download`
+    icon (middle-right) to download the signed `.apk` locally.
+  - After having this signed `.apk` downloaded locally, go ahead and click on the `Delete app bundle`
+    to delete the this app bundle from the list of available app bundles.
+- Install this download signed `.apk` using this command:
+  `adb install -r Downloads/wpandroid-deps-update-google-play-services-20-4-1-Signed.apk`
+  (note that you can use `./tools/rename_apk_aab.sh` to rename the download signed `.apk`)
+- If you were already logged-in to WordPress, log-out and log-in again using the `Continue with Google`
+  button (note that you need to use a non `A8C` account to be able to complete this flow).
+- Verify that the Google sign-in works, that you have successfully logged-in and are able to use the
+  app as expected.
+
+</details>
+
+-----
+
+### PlayServicesCodeScanner [[googlePlayServicesCodeScannerVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="playservicescodescanner"></a>
+
+<details>
+  <summary>1. [JP] Scan Login Code</summary>
+
+Step.1:
+- Build and install the `Jetpack` app (note that you don't need a release build, a debug build will
+  suffice).
+- Login to the `Jetpack` app with a `WP.com` account (note that you need to use a non `A8C` account
+  and a non `2FA` enabled account).
+- Navigate to the `Me` screen (click on avatar at top-right).
+- (STOP)
+
+Step.2:
+- Head over to your desktop and open a web browser (note that using an incognito tab works best).
+- Browse to `wordpress.com` (note that if you are logged-in, log-out first).
+- Tap the `Log In` link (top-right).
+- Tap the `Login via the mobile app` link in the list of options below the main `Continue` button
+  (bottom-middle).
+- Verify you are on the `Login via the mobile app` view and `Use QR Code to login` is shown, along
+  with a QR code for you to scan.
+- (STOP)
+
+Step.3:
+- Head back to your mobile.
+- Tap the `Scan Login Code` item on the `Me` screen you are currently at.
+- Scan the QR code on the web browser.
+- Follow the remaining prompts on your mobile to login to WordPress on your web browser (desktop),
```

Comment:
np: `to login to WordPress` → `to log in to WordPress`
Since "Log in" is the verb.

Reply:
[SAME](https://github.com/wordpress-mobile/WordPress-Android/pull/19552#discussion_r1389271813)